Dockerfile

```diff
@@ -34,7 +41,7 @@ RUN CGO_ENABLED=0 GOOS=linux GO111MODULE=on go build -ldflags="-s -w" -a -o mana
 
 # Use distroless as minimal base image to package the manager binary
 # Refer to https://github.com/GoogleContainerTools/distroless for more details
-FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
+FROM registry.access.redhat.com/ubi8/openjdk-11:latest
```

Comment:
@kabicin The [Red Hat page](https://catalog.redhat.com/software/containers/ubi8/openjdk-11/5dd6a4b45a13461646f677f4) states that `registry.access.redhat.com/ubi8/openjdk-11` has deprecated. So we shouldn't start using it. 

We'll be better off using Semeru runtime as we can influence that the images are refreshed and kept up to date. I suggest using Semeru 21 if possible (`icr.io/appcafe/ibm-semeru-runtimes:open-21-jre-ubi-minimal`) as 11 and 17 are expected to be out of support in 2027. Use UBI 8 minimal for now, but we can consider switching to UBI 9 in the future. 

If it's possible to copy the Semeru artifacts from the Semeru container image (as opposed to using it as the base), that's also worth considering.

Reply:
Updated to `icr.io/appcafe/ibm-semeru-runtimes:open-21-jre-ubi-minimal`